portality/formcontext/forms.py

```diff
@@ -156,7 +156,7 @@ class JournalInformation(Form):
     )
     keywords = TagListField('Add keyword(s) that best describe the journal (comma delimited)',
         [validators.DataRequired(), MaxLen(6, message='You can only enter up to {max_len} keywords.')],
-        description='Maximum 6. Keywords must be in English.'
+        description='Maximum 6. Keywords must be in English. All keywords are saved in lowercases.'
```

Comment:
Can we change the second sentence to `All keywords are stored in lowercase`

Reply:
@amdomanska just `lowercases` to `lowercase` still to do

Reply:
Sorry :/